### ISSUE TITLE
[Feature Request] Implement CrossQ

### ISSUE THREAD

**Issue by danielpalen** [Stable-Baselines-Team/stable-baselines3-contrib#238]
### 🚀 Feature

I would like to implement CrossQ (https://openreview.net/pdf?id=PczQtTsTIX) in SB3, as also suggested by @araffin (https://github.com/araffin/sbx/pull/36#issuecomment-2027392759), 

### Motivation

CrossQ is one of the current state-of-the-art deep reinforcement learning methods in terms of sample efficiency and substantionally more computationally efficefficient than the previous state-of-the-art (e.g. DroQ or REDQ), as it uses a low update-to-data ratio of 1. It is the first successful application of batch normalization within deep reinforcement learning, which is at the heart of it's efficiency. I think a PyTorch based reference implementation in SB3 would be very valuable for the research community.

### Pitch

As one of the first authors on the paper, I want to contribute a PyTorch based reference implementation of CrossQ to SB3, since the paper's implementation is in JAX.

### Checklist

- [X] I have checked that there is no similar [issue](https://github.com/Stable-Baselines-Team/stable-baselines3-contrib/issues) in the repo
- [X] If I'm requesting a new feature, I have proposed alternatives
